### PR TITLE
Add queue service example with producer and consumer.

### PR DESCRIPTION
**Pull request by bclews**
This pull request adds a new example demonstrating the use of the IVCAP Queue service. It includes a producer service that simulates publishing temperature data from buoys to a queue, and a consumer service that retrieves and logs the data from the queue. The example showcases a practical application of using queues in a distributed system for processing environmental data in the scientific domain.

### CODE REVIEW

File: queue/consumer/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,2 @@
+git+https://github.com/ivcap-works/ivcap-service-sdk-python@queue_svc#egg=ivcap_sdk_service
```
Comment (bclews) [inline queue/consumer/requirements.txt]: This is currently pinned to the `queue_svc` branch of the `ivcap-service-sdk-python`, which is ready for review. Once that branch gets merged into develop/main I will update these requirements files.

File: queue/producer/service.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+env IVCAP_SERVICE_ID=ivcap:service:e5fb1c5f-79ed-55bc-ab86-89aa44c8055c:queue-producer-service \
```
Comment (maxott) [inline queue/producer/service.yaml]: Why is this file here?

**Review by maxott — CHANGES_REQUESTED** [review]

File: queue/producer/service.yaml (hunk 2 above)
Comment (bclews) [reply queue/producer/service.yaml]: My carelessness! I will remove.

File: queue/consumer/requirements.txt (hunk 1 above)
Comment (bclews) [reply queue/consumer/requirements.txt]: Updated to point to `develop` now instead of `queue_svc`

File: queue/producer/service.yaml (hunk 2 above)
Comment (bclews) [reply queue/producer/service.yaml]: Updated.

**Comment by bclews** [comment]
Thanks for the review @maxott. 

The following commits removes the unnecessary queue producer service configuration and updates the requirements files to point to `ivcap-service-sdk-python@develop`:
- 6a7b93331fbdfb977c7ee780dec2415ada3a4b65
- bceb97eab90c4c61bf73c2c2362faf06d515cf5f

**Pull request merged**